### ISSUE TITLE
Show Commands Don't Work

### ISSUE THREAD

**Issue by avzero07** [Azure/SONiC#114]
Hey Team,

I have installed Sonic on an Arista 7060CX-32S. Followed the instructions from the quick start guide.

I have logged into sonic using the default username/password.

None of the **show** commands seem to work. The shell doesn't seem to recognize any of the show commands.

admin@sonic:~$ show ?
-bash: show: command not found

Am I missing something? Removed and reinstalled again but still no difference.

I have attached a screenshot too.

![screen shot 2017-10-05 at 3 47 02 pm](https://user-images.githubusercontent.com/27913879/31222373-7c316872-a9e4-11e7-8ecb-b631da25dd8f.png)

**Comment by jleveque** [comment]
This issue was fixed last night with https://github.com/Azure/sonic-buildimage/pull/1011

However, another issue with the 'show' command was exposed. It will be fixed once the following PR is merged: https://github.com/Azure/sonic-buildimage/pull/1013

After this PR is merged, please build a new image from the master branch and report back as to whether or not you experience any further issues.

**Comment by jleveque** [comment]
@avzero07: Azure/sonic-buildimage#1013 has been merged. Please verify when you have a moment. Thanks.

**Comment by avzero07** [comment]
@jleveque Hey I just tried the new image now and still got an error. Looks like there was an indent error in your script at line 226. :)

![screen shot 2017-10-06 at 8 29 21 am](https://user-images.githubusercontent.com/27913879/31261263-b5c05076-aa70-11e7-9328-4029c2674cf7.png)

Post the indent fix everything looks good.

![screen shot 2017-10-06 at 8 32 31 am](https://user-images.githubusercontent.com/27913879/31261311-f962e67c-aa70-11e7-850b-aec24cf3c514.png)

**Comment by jleveque** [comment]
Thanks for confirming. FYI, PR https://github.com/Azure/sonic-buildimage/pull/1018 has reverted things back to the way they were before this trouble began.
